examples/migrations/1111-task.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE films (
```

Comment:
Adicionar IF NOT EXISTS evitaria problema de o dev já ter criado a tabela no banco.
```suggestion
CREATE TABLE IF NOT EXISTS films (
```